python/pyspark/sql.py

```diff
@@ -1622,6 +1622,48 @@ def func(iterator):
             df = self._ssql_ctx.jsonRDD(jrdd.rdd(), scala_datatype)
         return DataFrame(df, self)
 
+    def load(self, path=None, dataSourceName=None, schema=None, **options):
+        """Returns the dataset specified by the data source and a set of options
+        as a DataFrame. An optional schema can be applied as the schema of returned
+        DataFrame. If dataSourceName is not provided, the default data source configured
+        by spark.sql.sources.default will be used.
+        """
+        if path is not None:
+            options["path"] = path
+        if dataSourceName is None:
+            dataSourceName = self._ssql_ctx.getConf("spark.sql.sources.default",
```

Comment:
It is actually trying to access SQL conf. Seems we do not have one in Python SQLContext. I will add getConf and setConf to it.